servers/curated-corpus-api/prisma/schema.prisma

```diff
@@ -129,6 +132,10 @@ model Section {
   deactivatedAt         DateTime?
   createdAt             DateTime        @default(now())
   updatedAt             DateTime        @updatedAt
+  // For custom sections, indicates when a section goes "live"
+  startDate             DateTime?       @db.Date
+  // For custom sections, indicates when a section "expires"
+  endDate               DateTime?       @db.Date
```

Comment:
Shouldn’t Prisma add the comments in the schema via the migration?

Reply:
I believe the comments are added manually -- prisma schema triggers the SQL migration